HerPublicWebsite/Views/Partials/LocalAuthorityMessages/NotParticipating/Default.cshtml

```diff
@@ -0,0 +1,8 @@
+@model HerPublicWebsite.Models.Questionnaire.NotParticipatingViewModel
+
+<p class="govuk-body">
+    Based on the information you provided, it looks like your local authority, 
+    <strong class="govuk-!-font-weight-bold">@Model.LocalAuthorityName</strong>,
+    is not signed up to use this service yet. 
```

Comment:
⚠️ The "yet" is incorrect.
The copy is just "is not signed up to use this service."

Reply:
This is the main difference between the old "Pending" state and the new "Not participating", so this one "yet" carries a lot of weight!

Reply:
I'm going to recopy all the copy text from the Jira ticket to be sure it's 1:1

Reply:
Great, thanks! I've checked as well, but would be good to get second eyes on it. 👍